stdlib/int64.gr

```diff
@@ -281,18 +520,37 @@ export let rec gte = (x: Int64, y: Int64) => {
   ret
 }
 
-// Int64 bit/logical operations
+/**
+ * @section Bitwise Logic: Boolean operations on the bits of Int64 values.
```

Comment:
```suggestion
 * @section Bitwise logic: Boolean operations on the bits of Int64 values.
```
Should logic be capitalized or not?

Reply:
It is a subheader so I think it would follow title casing, but other team members might have a different opinion.

Reply:
@spotandjake if you look at other multi-word sections, they don't use title casing. Let's keep it consistent.